src/ApolloHooksTypes.re

```diff
@@ -58,3 +60,9 @@ let errorPolicyToJs = errorPolicy =>
   | Ignore => "ignore"
   | All => "all"
   };
+
+module type Config = {
```

Comment:
This is defined already in reaons-apollo library (https://github.com/apollographql/reason-apollo/blob/master/src/ReasonApolloTypes.re). Maybe its not necessary defined here? Maybe ApolloHooksQuery and other modules should use from reason-apollo ?

Reply:
I don't know. It might be good to be independent of changes of the original module type. Actually I think the original type is defined in `graphql_ppx`. @fakenickels any thoughts.

Reply:
Exactly! We want to be apart from ReasonApollo in the consumer types, currently we are only sharing the ApolloClient bindings. Which we even want to have a separated shared binding exclusively for ApolloClient

Reply:
Understood, okey